RELEASE_NOTES.md

```diff
@@ -22,3 +22,5 @@
 - Fix `consumer_power()` not working certain configurations.
   In microgrids without consumers and no main meter, the formula
   would never return any values.
+- Fix `pv_power` not working in setups with 2 grid meters by using DFS
```

Comment:
I'd suggest to change `by using DFS` to something around `by using a new reliable function to search for components`